conf/nebula-graphd.conf.default

```diff
@@ -51,10 +51,11 @@
 --reuse_port=false
 # Backlog of the listen socket, adjust this together with net.core.somaxconn
 --listen_backlog=1024
-# Seconds before the idle connections are closed, 0 for never closed
---client_idle_timeout_secs=0
-# Seconds before the idle sessions are expired, 0 for no expiration
---session_idle_timeout_secs=0
+# The number of seconds Nebula service waits before closing the idle connections
+--client_idle_timeout_secs=28800
```

Comment:
28800 is 8 hours Is it a little long?

Reply:
28800 seconds is the default timeout MySQL uses.
Ref: https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_wait_timeout

Reply:
👍